### DISCUSSION TITLE
Issue warning if  WATCHTOWER_SCHEDULE has less than 6 parameters

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [containrrr/watchtower#2010] [Feature Requests]
Refering to [#2009](https://github.com/containrrr/watchtower/discussions/2009), it is not obvious to new years that the schedule parameter needs 6 parameters i.e. also seconds. A warning when this is not set would be useful.

**Comment by huyz** [comment]
@ZuluPilgrim you meant to file this under https://github.com/containrrr/watchtower/issues right?

**Comment by ZuluPilgrim** [comment]
I think so, I was asked to raise a feature request

André Basel
027 640 2527

Sent from Proton Mail Android

-------- Original Message --------

**Reply by huyz** [reply]
Yes, but you raised it in the **Discussions** section. I think it should be raised in the **Issues** section: https://github.com/containrrr/watchtower/issues

**Comment by ZuluPilgrim** [comment]
Ok, how do I move it?

André Basel
027 640 2527

Sent from Proton Mail Android

-------- Original Message --------

**Reply by huyz** [reply]
I don't think you can.  You would just need to create a new Issue
